gax-httpjson/src/main/java/com/google/api/gax/httpjson/ApiMessage.java

```diff
@@ -37,9 +37,12 @@
 
 @BetaApi
 public interface ApiMessage {
+  /* For a each fieldName in a set of fieldNames, fetch that field's List<String> value. */
```

Comment:
Better: "For each field name in fieldNames, ..."

Reply:
Done.